autorandr.py

```diff
@@ -33,6 +33,7 @@
 import subprocess
 import sys
 from distutils.version import LooseVersion as Version
+from copy import copy
```

Comment:
This overrides `import copy` from above, so ll `copy.deepcopy` calls below won't work anymore. Please use `copy.copy` instead.